ansible_collections/arista/cvp/plugins/modules/cv_configlet.py

```diff
@@ -19,6 +19,12 @@
 #
 
 from __future__ import absolute_import, division, print_function
+from ansible_collections.arista.cvp.plugins.module_utils import schema_v1 as schema
```

Comment:
Somehow thought there was also a rule about order of imports if they were built in vS non built in but I guess auto pep8 is happy this way?

Reply:
First configuration attempt was wrong. We need to ignore E402 - Fix module level import not at top of file in autopep8 to make ansible-test happy. Ansible requires imports to be after the documentation string...
Fixed in another commit. Imports stays were they are